code/game/objects/items/stacks/stack.dm

```diff
@@ -69,7 +69,7 @@
 
 /obj/item/stack/examine(mob/user, distance, is_adjacent)
 	. = ..()
-	if(distance <= 1)
+	if(is_adjacent)
```

Comment:
Be warned this is not equivalent. `is_adjacenf` checks for occlusion like being behind a window. `distance` does not.